### DISCUSSION TITLE
Postal server configurations change to base64

### DISCUSSION THREAD

**Discussion by AlexSprintCyber** [postalserver/postal#2687] [Help with using Postal]
When we are try to send the attachment from our mail and it seems this header is set and we are able to receive the file in email but it's corrupted.

It may be caused for the content-transfer-encoding is set to 7bit.   Is there a way to change it to base64 ?

**Comment by willpower232** [comment]
How are you constructing the message? Is the file definitely in its own part?

**Reply by AlexSprintCyber** [reply]
We are sending it via API key. Also uses postal under docker.

**Reply by willpower232** [reply]
Are you using a library to send via the API? are you using rawmessage or regular message? You aren't accidentally double base64-encoding are you?

**Reply by AlexSprintCyber** [reply]
We send it via postal. Just a regular message.

**Reply by willpower232** [reply]
how are you sending it via the API though? are you using one of the API libraries or doing it directly?
